quint/src/generated/.antlr/QuintLexer.java

```diff
@@ -1,4 +1,4 @@
-// Generated from /Users/thomas/src/quint/quint/src/generated/Quint.g4 by ANTLR 4.13.1
+// Generated from /Users/igor/devl/informal/quint/quint/src/generated/Quint.g4 by ANTLR 4.13.1
```

Comment:
oh, that's weird. Why does my HOME propagate in the source code :)

Reply:
Weird!